FAQ.md

```diff
@@ -420,6 +420,12 @@ applications:
         credentialFile: credentials.json
 ```
 
+You can either provide the instance connection name or provide the following variables to your application.
+
+    $ teresa app env-set DB_PROJECT=<gcp-project> --app <app-name>
```

Comment:
Is it possible to set just one of the variables? I don't know if my question makes sense, but different environment variables (usually) implies that it is possible to set them separately and independently.

Reply:
If this is not possible, what do you about using something like `GCP_CLOUDSQL_INSTANCE_NAME`?

Reply:
@RomuloOliveira made it. About if it is possible to set them independently, the cloudsql sidecar will likely break (and the deployment will surely fail). Do you think we can make some validations for that? Is that teresa's responsibility for making such validation?

Reply:
@vinicyusmacedo I don't know exactly, maybe we could just add it to the documentation saying that all three variables should be set in order to work or maybe we can use only the `GCP_CLOUDSQL_INSTANCE_NAME`.

I think it is more a design decision (and @drgarcia1986 could help us).